docker/README.md

```diff
@@ -0,0 +1,110 @@
+Docker Containerized Schema Registry
+====================================
+
+Dockerized Schema Registry application to run System tests. 
+
+
+# Dependencies
+
+- [Docker CE](https://www.docker.com/community-edition#/download)
+- [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
+- jq (brew install jq)
+
+# Usage
+
+### Starts the docker machine
+
+```
+sh kdc-registry.sh start-machine
+```
+It starts a Linux virtual machine setup and installs the Docker Engine on top of it.
+This VM is used as Docker host machine as there are known problems in Docker Engine when
+running it on Mac and Windows OS.
+
+<B>NOTE:</B> Once the machine started, you should set the environmental variables which configures the shell
+to execute the docker commands (docker / docker-compose) inside the VM.
+
+``` eval $(docker-machine env sr-host)```
+
+### Build the Registry Image
+```
+sh kdc-registry.sh build
+```
+
+It builds the Key Distribution Center, Apache Zookeeper, Apache Kafka and Schema Registry Images.
+And, pulls the official images of MySQL, Oracle and Postgresql database images from the docker store.
+
+To run registry application with Oracle db, user needs to manually download the [ojdbc.jar](http://www.oracle.com/technetwork/database/features/jdbc/jdbc-drivers-12c-download-1958347.html) from the Oracle website and 
+copy it to `extlibs` directory before building the image.
+
+To build Schema Registry from specific tag release, export this variable before building the image. (Only tar file supported)
+```
+export download_url="https://github.com/hortonworks/registry/releases/download/v0.5.0/hortonworks-registry-0.5.0.tar.gz"
```

Comment:
Can we change "download_url" to "schema_registry_download_url" ?

Reply:
Done.